test/thermodynamics/probability_of_ideal_gas_macrostate_test.py

```diff
@@ -0,0 +1,53 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import assert_equal, Quantity, units, errors
+from symplyphysics.core.symbols.probability import Probability
+from symplyphysics.laws.thermodynamics import probability_of_ideal_gas_macrostate as macrostate_law
+
+Args = namedtuple("Args", "p1 p2 p3 n1 n2 n3")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    p1 = Probability(0.1)
+    p2 = Probability(0.4)
+    p3 = Probability(0.5)
+    n1 = 10
+    n2 = 13
+    n3 = 5
+    return Args(p1=p1, p2=p2, p3=p3, n1=n1, n2=n2, n3=n3)
+
+
+def test_law(test_args: Args) -> None:
+    ps = test_args.p1, test_args.p2, test_args.p3
+    ns = test_args.n1, test_args.n2, test_args.n3
```

Comment:
Better use lists instead of tuples: `ns = [test_args.n1, test_args.n2, test_args.n3]`